README.md

```diff
@@ -21,6 +21,19 @@ To bind the java agent to a specific IP change the port number to `host:port`.
 
 See `./run_sample_httpserver.sh` for a sample script that runs the httpserver against itself.
 
+### Wildfly
```

Comment:
I don't think it's sane for us to document workarounds for every single application that's decided to go off the beaten path, there's enough inconsistency in JMX already for us to deal with.